docs/prefix-and-ip-target.md

```diff
@@ -9,26 +9,26 @@ When a new ENI is allocated, IPAMD will allocate either 1 prefix or number of pr
+|---------------|:-------------------:|:---------------:|:-------------------:|:----:|:----:|:------------:|:-----------------:|:---------------:|:----------------:|:---------:|
+| t3.small      |         1           |         -       |          -          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         1           |         -       |          -          |   5  |  3   |    1,2,2     |        4          |      1          |    2,1,1         |    59     |
+| t3.small      |         1           |         -       |          -          |  17  |  1   |     17       |        3          |      1          |      3           |    31     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| t3.small      |         -           |         1       |          1          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         -           |         1       |          1          |   5  |  3   |    1,2,2     |        3          |      0          |    1,1,1         |    43     |
+| t3.small      |         -           |         1       |          1          |  17  |  1   |     17       |        2          |      0          |      2           |    15     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| t3.small      |         -           |         2       |         10          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| t3.small      |         -           |         2       |         10          |   5  |  3   |    1,2,2     |        3          |      0          |    1,1,1         |    43     |
+| t3.small      |         -           |         2       |         10          |  17  |  1   |     17       |        2          |      0          |      2           |    15     |
+|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+| p3dn.24xlarge |         1           |         -       |          -          |   0  |  1   |      0       |        1          |      1          |      1           |    16     |
+| p3dn.24xlarge |         1           |         -       |          -          |   3  |  2   |    3,0       |        3          |      1          |    2,0           |    29     |
+| p3dn.24xlarge |         1           |         -       |          -          |  95  |  3   |   95,0,0     |        7          |      1          |    7,0,0         |    17     |
```

Comment:
If WARM_PREFIX_TARGET = 1 and all the prefixes including the warm prefix is from 1st ENI can we have 2 additional ENIs with no prefix allocated to them?

Reply:
Yeah you can have this until the next time reconciler starts. Next time reconciler will free up unused ENIs.

Reply:
I am assuming this table is for idle state after reconciliation has happened? So shouldn't the Prefixes per ENI be 7 instead of 7,0,0 and ENIs = 1

Reply:
Yeah after reconciliation it will become 7 prefixes and 1 ENI

Reply:
Synced with Jayanth offline. Later this can be modified to values where number of Pods remains the same and reconciliation has been done to reach a steady state.